packages/app-project/src/components/AuthModal/components/LoginForm/LoginFormContainer.js

```diff
@@ -0,0 +1,53 @@
+import { inject, observer } from 'mobx-react'
+import auth from 'panoptes-client/lib/auth'
+import { func, shape } from 'prop-types'
+import React, { Component } from 'react'
+
+import LoginForm from './LoginForm'
+
+@inject('store')
+@observer
+class LoginFormContainer extends Component {
+  constructor () {
+    super()
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onSubmit (values, { setFieldError, setSubmitting }) {
```

Comment:
We could add a test for the functionality of this handler this using nock.

Reply:
I've written tests for this using sinon - we're not testing the auth client, just that it's called correctly